wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,139 @@
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+
+- ![Beginner](/wiki/shared/diff/easy-s.png "Beginner") **Beginner**: in an ENHIX spread, it means the lowest level.
+- ![Advanced](/wiki/shared/diff/normal-s.png "Advanced") **Advanced**: in an ENHIX spread, it's a difficulty that is between Normal and Hard.
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") **Hyper**: in an ENHIX spread, it means a more difficult Hard (also known as Light Insane).
+- ![Extreme](/wiki/shared/diff/expert-s.png "Extreme") **Extreme**: uncommon, but it can be used to represent a difficulty tier above Extra/Expert.
+- ![Marathon](/wiki/shared/diff/expert-s.png "Marathon") **Marathon**: map that is over 5 minutes of [drain time](/wiki/drain_time).
+
+### ![osu!taiko](/wiki/shared/Taiko.gif "osu!taiko") osu!taiko
+
+- ![Hell Oni](/wiki/shared/diff/expert-t.png "Hell Oni") **Hell Oni**: commonly accepted as a difficulty that is a tier above Inner Oni / Ura Oni.
+
+### ![osu!catch](/wiki/shared/Ctb.gif "osu!catch") osu!catch
+
+- ![Deluge](/wiki/shared/diff/expert-c.png "Deluge") **Deluge**: used when there is a difficulty tier above Overdose.
+
+## Songs from other Rhythm Games
+
+This subsection shows the difficulty naming for mapsets of a song that comes from another rhythm game. Using said rhythm game's style of difficulty naming is acceptable.
+
+### Beatmania IIDX (or BMS songs)
+
+#### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+
+- ![Beginner](/wiki/shared/diff/easy-s.png "Beginner") Beginner
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") Hyper
+- ![Another](/wiki/shared/diff/insane-s.png "Another") Another
+- ![Black Another](/wiki/shared/diff/expert-s.png "Black Another") Black Another
+
+#### ![osu!mania](/wiki/shared/Mania.gif "osu!mania") osu!mania
+
+- ![Beginner](/wiki/shared/diff/easy-m.png "Beginner") Beginner 
+- ![Normal](/wiki/shared/diff/normal-m.png "Normal") Normal
+- ![Hyper](/wiki/shared/diff/hard-m.png "Hyper") Hyper
+- ![Another](/wiki/shared/diff/insane-m.png "Another") Another
+
+### SOUND VOLTEX (SDVX)
+
+Abbreviations on this particular difficulty naming scheme are acceptable (e.g. NOVICE → NOV).
+
+#### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+
+- ![BASIC](/wiki/shared/diff/easy-s.png "BASIC") BASIC 
+- ![NOVICE](/wiki/shared/diff/normal-s.png "NOVICE") NOVICE
+- ![ADVANCED](/wiki/shared/diff/hard-s.png "ADVANCED") ADVANCED
+- ![EXHAUST](/wiki/shared/diff/insane-s.png "EXHAUST") EXHAUST
+- ![INFINITE](/wiki/shared/diff/expert-s.png "INFINITE") INFINITE
+
+#### ![osu!mania](/wiki/shared/Mania.gif "osu!mania") osu!mania
+
+- ![NOV](/wiki/shared/diff/normal-m.png "NOV") NOV
+- ![ADV](/wiki/shared/diff/hard-m.png "ADV") ADV
+- ![EXH](/wiki/shared/diff/insane-m.png "EXH") EXH
+- ![INF](/wiki/shared/diff/expert-m.png "INF") INF
+
+### Pop 'n Music
+
+#### ![osu!mania](/wiki/shared/Mania.gif "osu!mania") osu!mania
```

Comment:
at this point the whole mode thing is just so ppl can keep track of it more easily? i mean it should maybe say somewhere that using this naming for any other mode is also acceptable

Reply:
this was a direct copy from the thread i kinda agree that it makes little sense, will rework something out of this. but having 4 icons per bullet point looks too hideous to me so i'll just use standard diff icons